site/_posts/2018-02-26-manageiq-survey.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Help Us Improve!
```

Comment:
Might be good to be a bit more specific in the title, perhaps "Quad Icons: Help Us Improve!"